.github/workflows/reusable-audit.yml

```diff
@@ -23,6 +23,8 @@ jobs:
         uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
         with:
           ref: ${{ matrix.ref }}
+      - name: Install tooling
+        uses: asdf-vm/actions/install@707e84f3ee349548310aeabdad0dd3bfcb9b69fa # v1.1.0
```

Comment:
This change does not properly take affect until after this Pull Request is merged as this re-usable workflow is always referenced from `main`. Hence, this check ("Check / Audit / Docker") will be failing.

Reply:
This was verified through the [continuous integration run](https://github.com/ericcornelissen/js-regex-security-scanner/actions/runs/3932854911/jobs/6725917231) for 0e58f1f95a924751749326a606df89b907491fb0